tiledb/ml/readers/tensorflow.py

```diff
@@ -13,18 +13,10 @@
 
 def TensorflowTileDBDataset(
     *all_array_params: ArrayParams,
-    batch_size: int,
-    shuffle_buffer_size: int = 0,
-    prefetch: int = tf.data.AUTOTUNE,
     num_workers: int = 0,
 ) -> tf.data.Dataset:
     """Return a tf.data.Dataset for loading data from TileDB arrays.
-
```

Comment:
The convention is to separate the docstring title from the parameters with an empty line.